pyvista/plotting/plotting.py

```diff
@@ -2138,6 +2139,13 @@ def close(self):
             except BaseException:
                 pass
 
+        for renderer in self.renderers:
+            renderer.camera_position = None
```

Comment:
We do not want to clear this - quite often users want to access the last used camera position after the plotter was closed